### ISSUE TITLE
Revive changes

### ISSUE THREAD

**Issue by jaraco** [jaraco/cssutils#1]
In the retirement of Bitbucket, some of the source code history was lost. In particular, the releases 1.0.1 and 1.0.2 have no associated history. Let's revive those changes in the repo.

**Comment by jaraco** [comment]
```
cssutils main $ git-id
b434370
cssutils main $ http --follow https://files.pythonhosted.org/packages/22/de/6b03e0088baf0299ab7d2e95a9e26c2092e9cb3855876b958b6a62175ca2/cssutils-1.0.1.tar.gz | tar xz --strip-components=1
cssutils main $ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   setup.cfg
        modified:   setup.py
        modified:   src/cssutils/__init__.py
        modified:   src/cssutils/css/cssstyledeclaration.py
        modified:   src/cssutils/profiles.py
        modified:   src/cssutils/tests/test_cssutils.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        PKG-INFO

no changes added to commit (use "git add" and/or "git commit -a")
cssutils main $ git commit -a -m "Apply changes from 1.0.1 release" --date 2015-10-11
[main ea13f5a] Apply changes from 1.0.1 release
 Date: Sun Oct 11 22:26:29 2015 -0400
 6 files changed, 12 insertions(+), 9 deletions(-)
cssutils main $ http --follow https://files.pythonhosted.org/packages/5c/0b/c5f29d29c037e97043770b5e7c740b6252993e4b57f029b3cd03c78ddfec/cssutils-1.0.2.tar.gz | tar xz --strip-components=1
cssutils main $ git add src setup.py
cssutils main $ git commit -a -m "Apply changes from 1.0.2 release." --date 2017-03-04
[main 59ecbb4] Apply changes from 1.0.2 release.
 Date: Sat Mar 4 21:27:45 2017 -0500
 81 files changed, 8789 insertions(+), 139 deletions(-)
 create mode 100644 src/cssutils/tests/sheets/096.css
 create mode 100644 src/cssutils/tests/sheets/097.css
 create mode 100644 src/cssutils/tests/sheets/1.css
 create mode 100644 src/cssutils/tests/sheets/1ascii.css
 create mode 100644 src/cssutils/tests/sheets/1import.css
 create mode 100644 src/cssutils/tests/sheets/1inherit-ascii.css
 create mode 100644 src/cssutils/tests/sheets/1inherit-iso.css
 create mode 100644 src/cssutils/tests/sheets/1inherit-utf8.css
 create mode 100644 src/cssutils/tests/sheets/1utf.css
 create mode 100644 src/cssutils/tests/sheets/2inherit-iso.css
 create mode 100644 src/cssutils/tests/sheets/2resolve.css
 create mode 100644 src/cssutils/tests/sheets/acid2.css
 create mode 100644 src/cssutils/tests/sheets/all.css
 create mode 100644 src/cssutils/tests/sheets/atrule.css
 create mode 100644 src/cssutils/tests/sheets/bad.css
 create mode 100644 src/cssutils/tests/sheets/basic.css
 create mode 100644 src/cssutils/tests/sheets/bundle.css
 create mode 100644 src/cssutils/tests/sheets/cases.css
 create mode 100644 src/cssutils/tests/sheets/csscombine-1.css
 create mode 100644 src/cssutils/tests/sheets/csscombine-2.css
 create mode 100644 src/cssutils/tests/sheets/csscombine-proxy.css
 create mode 100644 src/cssutils/tests/sheets/cthedot_default.css
 create mode 100644 src/cssutils/tests/sheets/default_html4.css
 create mode 100644 src/cssutils/tests/sheets/hacks.css
 create mode 100644 src/cssutils/tests/sheets/html.css
 create mode 100644 src/cssutils/tests/sheets/html20.css
 create mode 100644 src/cssutils/tests/sheets/html40.css
 create mode 100644 src/cssutils/tests/sheets/images/example.gif
 create mode 100644 src/cssutils/tests/sheets/import.css
 create mode 100644 src/cssutils/tests/sheets/import/images2/example2.gif
 create mode 100644 src/cssutils/tests/sheets/import/import-impossible.css
 create mode 100644 src/cssutils/tests/sheets/import/import2.css
 create mode 100644 src/cssutils/tests/sheets/import3.css
 create mode 100644 src/cssutils/tests/sheets/ll.css
 create mode 100644 src/cssutils/tests/sheets/ll2.css
 create mode 100644 src/cssutils/tests/sheets/multiple-values.css
 create mode 100644 src/cssutils/tests/sheets/page_test.css
 create mode 100644 src/cssutils/tests/sheets/sample_5.css
 create mode 100644 src/cssutils/tests/sheets/sample_7.css
 create mode 100644 src/cssutils/tests/sheets/simple.css
 create mode 100644 src/cssutils/tests/sheets/single-color.css
 create mode 100644 src/cssutils/tests/sheets/slashcode.css
 create mode 100644 src/cssutils/tests/sheets/t-HACKS.css
 create mode 100644 src/cssutils/tests/sheets/test-unicode.css
 create mode 100644 src/cssutils/tests/sheets/test.css
 create mode 100644 src/cssutils/tests/sheets/tigris.css
 create mode 100644 src/cssutils/tests/sheets/tigris2.css
 create mode 100644 src/cssutils/tests/sheets/u_simple.css
 create mode 100644 src/cssutils/tests/sheets/v_simple.css
 create mode 100644 src/cssutils/tests/sheets/var/start.css
 create mode 100644 src/cssutils/tests/sheets/var/use.css
 create mode 100644 src/cssutils/tests/sheets/var/vars.css
 create mode 100644 src/cssutils/tests/sheets/var/vars2.css
 create mode 100644 src/cssutils/tests/sheets/vars.css
 create mode 100644 src/cssutils/tests/sheets/varsimport.css
 create mode 100644 src/cssutils/tests/sheets/xhtml2.css
 create mode 100644 src/cssutils/tests/sheets/xhtml22.css
 create mode 100644 src/cssutils/tests/sheets/yuck.css
```

**Comment by jaraco** [comment]
Fixed in 59ecbb4.
